test/schedule/Schedule.integration.spec.ts

```diff
@@ -18,24 +18,29 @@ describe('Schedule', () => {
   let connection: Connection;
   let jobRepository: JobRepository;
   let mongoSchedule: MongoSchedule;
+  let secondMongoSchedule: MongoSchedule;
```

Comment:
Do we only need the second schedule to prove that two schedules can coexist? It doesn't seem to really be used in the tests.